ui/app/registration/controllers/patientCommonController.js

```diff
@@ -17,7 +17,153 @@ angular.module('bahmni.registration')
             $scope.readOnlyExtraIdentifiers = appService.getAppDescriptor().getConfigValue("readOnlyExtraIdentifiers");
             $scope.showSaveConfirmDialogConfig = appService.getAppDescriptor().getConfigValue("showSaveConfirmDialog");
             $scope.showSaveAndContinueButton = false;
-            $scope.moduleName = appService.getAppDescriptor().getConfigValue('registrationModuleName');
+            $scope.regExtPoints = appService.getAppDescriptor().getExtensions("org.bahmni.registration.identifier", "link");
+
+            $scope.showIframe = false;
+            var buttonMap = new Map();
```

Comment:
Can we think of better name? if this is storing identifier verification to extensions - "identifierExtnMap"